src/prashantCJune20/SumofIntinString.java

```diff
@@ -0,0 +1,19 @@
+package prashantCJune20;
+
+public class SumofIntinString {
+	
+	void addDigits(String input) {
+		int addition = 0;
+		for(int index = 0;index<input.length();index++) {
+			if(Character.isDigit(input.charAt(index)))
+				addition = addition	+ Integer.parseInt(String.valueOf(input.charAt(index)));
```

Comment:
Inside the If block instead this long statement 
```java 
addition = addition	+ Integer.parseInt(String.valueOf(input.charAt(index)));
```

You could simply do 
```java 
addition += input.charAt(index)-'0';
```

Reply:
Logic changed as per comments.